frontend/src/lib/components/apps/components/helpers/RunnableComponent.svelte

```diff
@@ -61,7 +61,10 @@
 	$: fields && (currentStaticValues = computeStaticValues())
 	$: if (JSON.stringify(currentStaticValues) != JSON.stringify(lazyStaticValues)) {
 		lazyStaticValues = currentStaticValues
-		setDebouncedExecute()
+
+		if (autoRefresh) {
```

Comment:
@rubenfiszel A button or a form (autorefresh = false) would recompute automatically, and when for a example an user input was missing, an error was thrown when the app was mounted.